test/requirements-test.txt

```diff
@@ -9,3 +9,4 @@ openvino-dev
 psutil
 pytorch_lightning
 torchvision
+neural-compressor
```

Comment:
nit: can you add this in ascending order?

Reply:
Fixed. Thanks for your review.